src/main/java/org/reactome/release/qa/check/NewRegulationChecker.java

```diff
@@ -0,0 +1,84 @@
+package org.reactome.release.qa.check;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.reactome.release.qa.annotations.SliceQATest;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QACheckerHelper;
+import org.reactome.release.qa.common.QAReport;
+
+/**
+ *  This QA check finds any ReactionlikeEvents that have received a new, unreviewed Regulation instance (via 'regulatedBy' attribute).
+ * @author jcook
+ */
+
+import java.util.*;
+
+@SliceQATest
+public class NewRegulationChecker extends AbstractQACheck implements ChecksTwoDatabases
+{
+
+    private MySQLAdaptor priorAdaptor;
+
+    @Override
+    public String getDisplayName()
+    {
+        // Output report file name.
+        return "ReactionlikeEvent_Regulations_Not_Reviewed";
+    }
+
+    @Override
+    public void setOtherDBAdaptor(MySQLAdaptor adaptor)
+    {
+        this.priorAdaptor = adaptor;
+    }
+
+    @Override
+    public QAReport executeQACheck() throws Exception
+    {
+        QAReport report = new QAReport();
+        report.setColumnHeaders("DBID","DisplayName", "SchemaClass", "Issue", "MostRecentAuthor");
+        // Find all ReactionlikeEvents that have a filled 'regulatedBy' attribute in current slice.
+        Set<GKInstance> currentRlEsWithRegulations = getRlEsWithRegulationInstances(dba);
+
+        for (GKInstance currentRlE : currentRlEsWithRegulations) {
+            GKInstance previousRlE = priorAdaptor.fetchInstance(currentRlE.getDBID());
+            // QA check
+            if (newRegulationWithoutReviewed(currentRlE, previousRlE)){
+                report.addLine(currentRlE.getDBID().toString(), currentRlE.getDisplayName(), currentRlE.getSchemClass().getName(), "ReactionlikeEvent with new regulatedBy instance but has not yet been reviewed", QACheckerHelper.getLastModificationAuthor(currentRlE));
```

Comment:
Maybe for readability, it might be helpful to have each column value on its own line.  You could do something like:

```
report.addLine(
    currentRlE.getDBID().toString(),
    currentRlE.getDisplayName(),
    ...
);
```